### ISSUE TITLE
Why isn't createTandem used in this part of ParallelCircuit?

### ISSUE THREAD

**Issue by samreid** [phetsims/capacitor-lab-basics#83]
During #79 I noticed that ParallelCircuit has this code:

```js
  var createWires = function( config, battery, lightBulbs, capacitors,
                             circuitSwitches, circuitConnectionProperty, tandem ) {

    var wires = [];

    // wire battery to first capacitor, there must be at least one capacitor in the circuit
    assert && assert( capacitors.length > 0, 'There must be at least one capacitor in the circuit' );
    wires.push( BatteryToSwitchWire.BatteryToSwitchWireTop(
      config,
      battery,
      capacitors[ 0 ].topCircuitSwitch,
      tandem
    ) );
    wires.push( BatteryToSwitchWire.BatteryToSwitchWireBottom(
      config,
      battery,
      capacitors[ 0 ].bottomCircuitSwitch,
      tandem
    ) );
```

Shouldn't tandem.createTandem be used in these calls?  If not, why not?

**Comment by andrewadare** [comment]
`tandem.createTandem` is invoked inside the BatteryToSwitchWire factory methods. We got away with it because they were called once per context, so there were no Tandem name collisions.

Because the factory methods are simply thin wrappers around the BatteryToSwitchWire constructor, and all the parameters are conveniently available at the call site, I tried to "close down" the factories and call the constructors directly. This resulted in a strange layout bug:
![capacitor lab- basics screenshot](https://cloud.githubusercontent.com/assets/154163/15187118/15b9ee90-175d-11e6-9ddd-585bdb011494.png)

Perhaps the `createWires` callback returns its wires array at a different time than before, somehow changing the rendering order. This seems bad: pulling on this small thread started to unravel the sweater.

But rather than troubleshooting this, I think I should move the createTandem to the call site and move on.
